src/style.css

```diff
@@ -1,3 +1,96 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media all and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+@media all and (min-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+.logo_img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin: 10px 0 10px 50px;
```

Comment:
No need to add margin to logo, you have align-items: center rule to center your items vertically, and no need to add margin-left: 50px to logo and to list, just add padding: 0 50px to header